src/corner_kick/src/__tests__/example.ts

```diff
@@ -0,0 +1,3 @@
+test('example jest test', () => {
```

Comment:
Missing function comment describing test

Reply:
Added a function about how it is a sanity test to check if Jest is configured.